tensorflow/core/util/mkl_util.h

```diff
@@ -103,6 +103,10 @@ typedef enum {
   Dim3d_I = 1
 } MklDnnDims3D;
 
+enum class MklQuantization {
```

Comment:
Could you please add a comment explaining what this enum means.

Reply:
@tatianashp Do you mean MklDnnDims3D or MklQuantization?

Reply:
@tatianashp Added comments for enums.